ifrs17/Import/Importers.ipynb

```diff
@@ -593,6 +571,100 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Validations"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Validation for Active Data Node States"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public async static Task ValidateForDataNodeStateActiveAsync<T>(this IWorkspace workspace, Dictionary<string, DataNodeData> dataNodes) where T : BaseDataRecord",
+                "\n{   ",
+                "\n    foreach(var item in (await workspace.Query<T>().ToArrayAsync()).GroupBy(x => x.DataNode))",
+                "\n        if(!dataNodes.ContainsKey(item.First().DataNode))",
+                "\n            ApplicationMessage.Log(Error.InactiveDataNodeState, item.First().DataNode);",
+                "\n}",
+                "\n"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Validate for Data Node States Logic"
```

Comment:
This is repeated. I guess come from the merge conflict resolution. Have a look overall at the file to spot any other mistake like this one